client/questions_answers/QuestionsAnswers.jsx

```diff
@@ -106,7 +112,15 @@ class QuestionsAnswers extends React.Component {
         showMoreAnsweredQuestionsButton: true
       });
     } else {
-      let searchResults = this.state.renderedQuestions.filter((question) => question.question_body.includes(query));
+      let searchResults = JSON.parse(JSON.stringify(this.state.renderedQuestions));
+      searchResults = searchResults.filter((question) => question.question_body.includes(query));
+
+      searchResults.forEach((question) => {
+        question.question_body = (
+          <span dangerouslySetInnerHTML={{__html: question.question_body.split(query).join(`<mark>${query}</mark>`)}}></span>
```

Comment:
Haha is that a real label attribute or something you came up with?

Reply:
lol. totally real in react!